java.lang.NullPointerException  in isAnimating method

we found a lot of crashes in production for nullPointer,
and this is the exception's stacktrace:

java.lang.NullPointerException
com.airbnb.lottie.LottieDrawable.isAnimating
`java.lang.RuntimeException: 
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:2326)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:2387)
  at android.app.ActivityThread.access$800 (ActivityThread.java:147)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1281)
  at android.os.Handler.dispatchMessage (Handler.java:102)
  at android.os.Looper.loop (Looper.java:135)
  at android.app.ActivityThread.main (ActivityThread.java:5264)
  at java.lang.reflect.Method.invoke (Method.java)
  at java.lang.reflect.Method.invoke (Method.java:372)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:900)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:695)
Caused by: android.view.InflateException: 
  at android.view.LayoutInflater.createView (LayoutInflater.java:633)
  at co.lokalise.android.sdk.core.LokaliseLayoutInflater.createCustomViewInternal (LokaliseLayoutInflater.java:233)
  at co.lokalise.android.sdk.core.LokaliseLayoutInflater.access$000 (LokaliseLayoutInflater.java:26)
  at co.lokalise.android.sdk.core.LokaliseLayoutInflater$PrivateWrapperFactory2.onCreateView (LokaliseLayoutInflater.java:361)
  at android.view.LayoutInflater$FactoryMerger.onCreateView (LayoutInflater.java:177)
  at android.view.LayoutInflater.createViewFromTag (LayoutInflater.java:733)
  at android.view.LayoutInflater.rInflate (LayoutInflater.java:806)
  at android.view.LayoutInflater.rInflate (LayoutInflater.java:809)
  at android.view.LayoutInflater.rInflate (LayoutInflater.java:809)
  at android.view.LayoutInflater.inflate (LayoutInflater.java:504)
  at co.lokalise.android.sdk.core.LokaliseLayoutInflater.inflate (LokaliseLayoutInflater.java:84)
  at android.view.LayoutInflater.inflate (LayoutInflater.java:414)
  at m.c.a.Fragment.onCreateView (m.c.a.Fragment.java:105)
  at androidx.fragment.app.Fragment.performCreateView (Fragment.java:2612)
  at androidx.fragment.app.FragmentManagerImpl.moveToState (FragmentManagerImpl.java:874)
  at androidx.fragment.app.FragmentManagerImpl.moveFragmentToExpectedState (FragmentManagerImpl.java:1228)
  at androidx.fragment.app.FragmentManagerImpl.moveToState (FragmentManagerImpl.java:1293)
  at androidx.fragment.app.FragmentManagerImpl.dispatchStateChange (FragmentManagerImpl.java:2646)
  at androidx.fragment.app.FragmentManagerImpl.dispatchActivityCreated (FragmentManagerImpl.java:2600)
  at androidx.fragment.app.FragmentController.dispatchActivityCreated (FragmentController.java:246)
  at androidx.fragment.app.FragmentActivity.onStart (FragmentActivity.java:542)
  at androidx.appcompat.app.AppCompatActivity.onStart (AppCompatActivity.java:201)
  at android.app.Instrumentation.callActivityOnStart (Instrumentation.java:1220)
  at android.app.Activity.performStart (Activity.java:5991)
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:2279)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:2387)
  at android.app.ActivityThread.access$800 (ActivityThread.java:147)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1281)
  at android.os.Handler.dispatchMessage (Handler.java:102)
  at android.os.Looper.loop (Looper.java:135)
  at android.app.ActivityThread.main (ActivityThread.java:5264)
  at java.lang.reflect.Method.invoke (Method.java)
  at java.lang.reflect.Method.invoke (Method.java:372)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:900)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:695)
Caused by: java.lang.reflect.InvocationTargetException: 
  at java.lang.reflect.Constructor.newInstance (Constructor.java)
  at java.lang.reflect.Constructor.newInstance (Constructor.java:288)
  at android.view.LayoutInflater.createView (LayoutInflater.java:607)
  at co.lokalise.android.sdk.core.LokaliseLayoutInflater.createCustomViewInternal (LokaliseLayoutInflater.java:233)
  at co.lokalise.android.sdk.core.LokaliseLayoutInflater.access$000 (LokaliseLayoutInflater.java:26)
  at co.lokalise.android.sdk.core.LokaliseLayoutInflater$PrivateWrapperFactory2.onCreateView (LokaliseLayoutInflater.java:361)
  at android.view.LayoutInflater$FactoryMerger.onCreateView (LayoutInflater.java:177)
  at android.view.LayoutInflater.createViewFromTag (LayoutInflater.java:733)
  at android.view.LayoutInflater.rInflate (LayoutInflater.java:806)
  at android.view.LayoutInflater.rInflate (LayoutInflater.java:809)
  at android.view.LayoutInflater.rInflate (LayoutInflater.java:809)
  at android.view.LayoutInflater.inflate (LayoutInflater.java:504)
  at co.lokalise.android.sdk.core.LokaliseLayoutInflater.inflate (LokaliseLayoutInflater.java:84)
  at android.view.LayoutInflater.inflate (LayoutInflater.java:414)
  at it.m.c.a.Fragment.onCreateView (m.c.a.Fragment.java:105)
  at androidx.fragment.app.Fragment.performCreateView (Fragment.java:2612)
  at androidx.fragment.app.FragmentManagerImpl.moveToState (FragmentManagerImpl.java:874)
  at androidx.fragment.app.FragmentManagerImpl.moveFragmentToExpectedState (FragmentManagerImpl.java:1228)
  at androidx.fragment.app.FragmentManagerImpl.moveToState (FragmentManagerImpl.java:1293)
  at androidx.fragment.app.FragmentManagerImpl.dispatchStateChange (FragmentManagerImpl.java:2646)
  at androidx.fragment.app.FragmentManagerImpl.dispatchActivityCreated (FragmentManagerImpl.java:2600)
  at androidx.fragment.app.FragmentController.dispatchActivityCreated (FragmentController.java:246)
  at androidx.fragment.app.FragmentActivity.onStart (FragmentActivity.java:542)
  at androidx.appcompat.app.AppCompatActivity.onStart (AppCompatActivity.java:201)
  at android.app.Instrumentation.callActivityOnStart (Instrumentation.java:1220)
  at android.app.Activity.performStart (Activity.java:5991)
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:2279)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:2387)
  at android.app.ActivityThread.access$800 (ActivityThread.java:147)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1281)
  at android.os.Handler.dispatchMessage (Handler.java:102)
  at android.os.Looper.loop (Looper.java:135)
  at android.app.ActivityThread.main (ActivityThread.java:5264)
  at java.lang.reflect.Method.invoke (Method.java)
  at java.lang.reflect.Method.invoke (Method.java:372)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:900)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:695)
Caused by: java.lang.NullPointerException: 
  at com.airbnb.lottie.LottieDrawable.isAnimating (LottieDrawable.java:732)
  at com.airbnb.lottie.LottieAnimationView.isAnimating (LottieAnimationView.java:613)
  at com.airbnb.lottie.LottieAnimationView.onVisibilityChanged (LottieAnimationView.java:246)
  at android.view.View.dispatchVisibilityChanged (View.java:8643)
  at android.view.View.setFlags (View.java:9686)
  at android.view.View.<init> (View.java:4146)
  at android.widget.ImageView.<init> (ImageView.java:139)
  at android.widget.ImageView.<init> (ImageView.java:135)
  at androidx.appcompat.widget.AppCompatImageView.<init> (AppCompatImageView.java:72)
  at androidx.appcompat.widget.AppCompatImageView.<init> (AppCompatImageView.java:68)
  at com.airbnb.lottie.LottieAnimationView.<init> (LottieAnimationView.java:91)
  at com.airbnb.lottie.LottieAnimationView.<init> (LottieAnimationView.java:91)
  at java.lang.reflect.Constructor.newInstance (Constructor.java)
  at java.lang.reflect.Constructor.newInstance (Constructor.java:288)
  at android.view.LayoutInflater.createView (LayoutInflater.java:607)
  at co.lokalise.android.sdk.core.LokaliseLayoutInflater.createCustomViewInternal (LokaliseLayoutInflater.java:233)
  at co.lokalise.android.sdk.core.LokaliseLayoutInflater.access$000 (LokaliseLayoutInflater.java:26)
  at co.lokalise.android.sdk.core.LokaliseLayoutInflater$PrivateWrapperFactory2.onCreateView (LokaliseLayoutInflater.java:361)
  at android.view.LayoutInflater$FactoryMerger.onCreateView (LayoutInflater.java:177)
  at android.view.LayoutInflater.createViewFromTag (LayoutInflater.java:733)
  at android.view.LayoutInflater.rInflate (LayoutInflater.java:806)
  at android.view.LayoutInflater.rInflate (LayoutInflater.java:809)
  at android.view.LayoutInflater.rInflate (LayoutInflater.java:809)
  at android.view.LayoutInflater.inflate (LayoutInflater.java:504)
  at co.lokalise.android.sdk.core.LokaliseLayoutInflater.inflate (LokaliseLayoutInflater.java:84)
  at android.view.LayoutInflater.inflate (LayoutInflater.java:414)
  at m.c.a.Fragment.onCreateView (m.c.a.Fragment.java:105)
  at androidx.fragment.app.Fragment.performCreateView (Fragment.java:2612)
  at androidx.fragment.app.FragmentManagerImpl.moveToState (FragmentManagerImpl.java:874)
  at androidx.fragment.app.FragmentManagerImpl.moveFragmentToExpectedState (FragmentManagerImpl.java:1228)
  at androidx.fragment.app.FragmentManagerImpl.moveToState (FragmentManagerImpl.java:1293)
  at androidx.fragment.app.FragmentManagerImpl.dispatchStateChange (FragmentManagerImpl.java:2646)
  at androidx.fragment.app.FragmentManagerImpl.dispatchActivityCreated (FragmentManagerImpl.java:2600)
  at androidx.fragment.app.FragmentController.dispatchActivityCreated (FragmentController.java:246)
  at androidx.fragment.app.FragmentActivity.onStart (FragmentActivity.java:542)
  at androidx.appcompat.app.AppCompatActivity.onStart (AppCompatActivity.java:201)
  at android.app.Instrumentation.callActivityOnStart (Instrumentation.java:1220)
  at android.app.Activity.performStart (Activity.java:5991)
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:2279)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:2387)
  at android.app.ActivityThread.access$800 (ActivityThread.java:147)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1281)
  at android.os.Handler.dispatchMessage (Handler.java:102)
  at android.os.Looper.loop (Looper.java:135)
  at android.app.ActivityThread.main (ActivityThread.java:5264)
  at java.lang.reflect.Method.invoke (Method.java)
  at java.lang.reflect.Method.invoke (Method.java:372)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:900)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:695)`


, is it possible to find a solution?

--- a/lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java
+++ b/lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java
@@ -753,6 +753,11 @@ public class LottieDrawable extends Drawable implements Drawable.Callback, Anima
   }
 
   public boolean isAnimating() {
+    // On some versions of Android, this is called from the LottieAnimationView constructor, before animator was created.
+    // https://github.com/airbnb/lottie-android/issues/1430
+    if (animator == null) {
+      return false;
+    }
     return animator.isRunning();
   }
 

